github-sync/github-data/sigstore/repositories.yaml

```diff
@@ -1410,6 +1410,66 @@ repositories:
         dismissalRestrictions:
           - tuf-root-signing-codeowners
           - sigstore-keyholders
+  - name: root-signing-staging
+    owner: sigstore
+    description: "Staging TUF repository for Sigstore trust root"
+    homepageUrl: ""
+    defaultBranch: main
+    allowAutoMerge: false
+    allowMergeCommit: true
+    allowRebaseMerge: false
+    allowSquashMerge: false
+    archived: false
+    autoInit: false
+    deleteBranchOnMerge: false
+    hasDownloads: false
+    hasIssues: true
+    hasProjects: false
+    hasWiki: false
+    vulnerabilityAlerts: true
+    visibility: public
+    licenseTemplate: ""
+    topics: []
+    collaborators:
+      - username: sigstore-bot
+        permission: push
+      - username: sigstore-review-bot
+        permission: push
+    teams:
+      - name: tuf-root-signing-staging-codeowners
+        permission: maintain
```

Comment:
for this we will need to do in a follow up because we need to know the id for the team